maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -920,6 +920,20 @@ data class ToggleAirplaneModeCommand(
     }
 }
 
+data class InstallApplicationCommand(
```

Comment:
let's rename it to `InstallAppCommand` (for consistency with existing `LaunchAppCommand`)